### ISSUE TITLE
VSCode fails to load after install

### ISSUE THREAD

**Issue by polygonnedpotato** [KatsuteDev/Background#303]
### Operating System

Debian Sid (Linux 6.7.9 amd64)

### VSCode Version

1.88.0-insider

### Extension Version

2.10.0

### Settings

```json
{
    "background.windowBackgrounds": [
        "redacted"
    ],
    "background.autoInstall": false,
    "background.renderContentAboveBackground": true,
    "background.smoothImageRendering": true
}
```

### Issue

after installing, VSCode runs into this exception: `[69550:0322/205531.622010:INFO:CONSOLE(2933)] "Uncaught SyntaxError: Unexpected identifier 'Oooooo'", source: vscode-file://vscode-app/home/kitsune/Applications/VSCode-linux-x64/resources/app/out/vs/workbench/workbench.desktop.main.js (2933)`.

### How to replicate

1. Install latest insider build of VSCode.
2. Install background patch.
3. watch as vscode does literally nothing.

**Comment by Katsute** [comment]
This might be caused by a bad image path, do your image paths contain any symbols?

**Comment by polygonnedpotato** [comment]
no, they shouldn't

**Comment by Katsute** [comment]
This should be fixed now, update to v2.10.1

**Comment by polygonnedpotato** [comment]
thanks
